packages/salesforcedx-vscode-core/package.json

```diff
@@ -803,6 +803,11 @@
           "type": "boolean",
           "default": false,
           "description": "%experimental_deploy_retrieve_description%"
+        },
```

Comment:
You shouldn't need to add the flag bc it was already added through a different PR. If you rebase from develop, you should see the changes!